analysisservice/src/test/java/nu/yona/server/analysis/service/AnalysisEngineServiceTests.java

```diff
@@ -264,30 +311,29 @@ public void messageCreatedOnMatch()
 		Set<String> conflictCategories = new HashSet<>(Arrays.asList("lotto"));
 		service.analyze(userAnonId, new NetworkActivityDto(conflictCategories, "http://localhost/test", Optional.empty()));
 
-		// Verify that there is a day activity update.
-		// first the DayActivity is created and then updated with the Activity
-		// so (create, update) = 2 times save
-		ArgumentCaptor<DayActivity> dayActivity = ArgumentCaptor.forClass(DayActivity.class);
-		verify(mockDayActivityRepository, times(2)).save(dayActivity.capture());
-		Activity activity = dayActivity.getValue().getLastActivity();
-		assertThat("Expect right user set to activity", dayActivity.getValue().getUserAnonymized(), equalTo(userAnonEntity));
+		// Verify there is a an activity in a day activity inside a week activity
+		ArgumentCaptor<UserAnonymized> userAnonymizedCaptor = ArgumentCaptor.forClass(UserAnonymized.class);
+		verify(mockUserAnonymizedService, atLeastOnce()).updateUserAnonymized(userAnonymizedCaptor.capture());
```

Comment:
You can write it as `verify(mockUserAnonymizedService, atLeastOnce()).updateUserAnonymized(userAnonEntity);`, isn't it?